Thrive.csproj

```diff
@@ -21,6 +21,8 @@
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="simulation_parameters\CompoundLoader.cs" />
+    <Compile Include="src\engine\CheatManager.cs" />
+    <Compile Include="src\engine\CheatMenu.cs" />
```

Comment:
I think the CheatMenu.cs should be thrown in GUI common folder, the CheatManager file can stay in the engine folder.